### ISSUE TITLE
Cannot find checkpoint in eval_low_res_given_prompt.py

### ISSUE THREAD

**Issue by zheminzhang96** [batmanlab/MedSyn#1]
I tried to load the trained model weights obtained from your Google drive in trainer.load(-1) eval_low_res_given_prompt.py. But got the following errors: 
![error_msg](https://github.com/user-attachments/assets/aa7f3246-3a89-45e4-a1d3-e427e10b0218)

**Comment by CugAgo** [comment]
Have you solved this problem？

**Comment by zheminzhang96** [comment]
> Have you solved this problem？

yes.

**Comment by CugAgo** [comment]
I think in the load function, it is necessary to load the ckpt weight file, but the downloaded file is in pin format. Have you modified the code? How do you read the weights？
Thank you for your reply.

**Comment by zheminzhang96** [comment]
> I think in the load function, it is necessary to load the ckpt weight file, but the downloaded file is in pin format. Have you modified the code? How do you read the weights？ Thank you for your reply.

Yes, I modified code. I provided the .bin path as _checkpoint_path_. Then used torch.load(...) instead of accelerator.load_state(...) 
![medsyn_load](https://github.com/user-attachments/assets/3bdb767f-cbde-43b3-a947-9a2b648562bb)

**Comment by CugAgo** [comment]
> > I think in the load function, it is necessary to load the ckpt weight file, but the downloaded file is in pin format. Have you modified the code? How do you read the weights？ Thank you for your reply.
> 
> Yes, I modified code. I provided the .bin path as _checkpoint_path_. Then used torch.load(...) instead of accelerator.load_state(...) ![medsyn_load](https://private-user-images.githubusercontent.com/61330200/400876879-3bdb767f-cbde-43b3-a947-9a2b648562bb.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMzA2MDYsIm5iZiI6MTczNjMzMDMwNiwicGF0aCI6Ii82MTMzMDIwMC80MDA4NzY4NzktM2JkYjc2N2YtY2JkZS00M2IzLWE5NDctOWEyYjY0ODU2MmJiLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDA5NTgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0ODc3OTdiMjQ3NGMzNzhmYjM1ZTcwY2E4OWY5YTQ4NjlmMDYyNjMyMGUyZGFlNjY3ZGIxNTUzYzUwODJjZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5l0ftzg5t1lSkF3el96Mufo5_yzRfAkMu4mODkbUlwA)

If use ema_model.load_state_dict() instead of accelerator.load_state(...). Does the accelerator in the model still need to load parameters? Do the accelerators in the model still need to load parameters, or do it also need to load the checkpoint？
![image](https://github.com/user-attachments/assets/19d629de-c948-4108-80d8-055982440ce5)

Thank you！

**Comment by CugAgo** [comment]
> > > I think in the load function, it is necessary to load the ckpt weight file, but the downloaded file is in pin format. Have you modified the code? How do you read the weights？ Thank you for your reply.
> > 
> > 
> > Yes, I modified code. I provided the .bin path as _checkpoint_path_. Then used torch.load(...) instead of accelerator.load_state(...) ![medsyn_load](https://private-user-images.githubusercontent.com/61330200/400876879-3bdb767f-cbde-43b3-a947-9a2b648562bb.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMzA2MDYsIm5iZiI6MTczNjMzMDMwNiwicGF0aCI6Ii82MTMzMDIwMC80MDA4NzY4NzktM2JkYjc2N2YtY2JkZS00M2IzLWE5NDctOWEyYjY0ODU2MmJiLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDA5NTgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0ODc3OTdiMjQ3NGMzNzhmYjM1ZTcwY2E4OWY5YTQ4NjlmMDYyNjMyMGUyZGFlNjY3ZGIxNTUzYzUwODJjZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5l0ftzg5t1lSkF3el96Mufo5_yzRfAkMu4mODkbUlwA)
> 
> If use ema_model.load_state_dict() instead of accelerator.load_state(...). Does the accelerator in the model still need to load parameters? Do the accelerators in the model still need to load parameters, or do it also need to load the checkpoint？ ![image](https://private-user-images.githubusercontent.com/75002743/401110557-19d629de-c948-4108-80d8-055982440ce5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzNDU4NzIsIm5iZiI6MTczNjM0NTU3MiwicGF0aCI6Ii83NTAwMjc0My80MDExMTA1NTctMTlkNjI5ZGUtYzk0OC00MTA4LTgwZDgtMDU1OTgyNDQwY2U1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDE0MTI1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY3MDc4ODczYjZkZmUyM2UzOWVmMDZlMDA0YTM0N2U5OTIwOGRjYzFmZDcxYzllY2U3MDM1ZGI4OTQwMjQxMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.KRx5cXRcZS3NEDBUE4XUD5JueUXvgFFIusPaHKXcg38)
> 
> Thank you！

I have solved this problem and it seems that there is no need to load parameters for accelerator, so it can run correctly. Thank you

**Comment by qianxingyucode** [comment]
@CugAgo @zheminzhang96  Hi, have you ever generated successfully? I try to generate with the ckpt in the repo, but get bad generation. 

![Image](https://github.com/user-attachments/assets/7329e9c7-71a2-40e2-b925-325517351542)

**Comment by Treeboy2762** [comment]
@qianxingyucode Hi, were you able to solve the issue? Maybe I can help you troubleshoot it

<img width="643" alt="Image" src="https://github.com/user-attachments/assets/079299b7-6212-4c40-805c-74d69ba10205" />

**Comment by qianxingyucode** [comment]
@Treeboy2762 Yes, I really hope you can help me to figure out the reason why the generation is not good. Could you please share some details about your reproduction? How can I contact you? Could you please email me xingq004@gmail.com

**Comment by Mr-Ma-yikun** [comment]
> [@CugAgo](https://github.com/CugAgo) [@zheminzhang96](https://github.com/zheminzhang96) Hi, have you ever generated successfully? I try to generate with the ckpt in the repo, but get bad generation.
> 
> ![Image](https://github.com/user-attachments/assets/7329e9c7-71a2-40e2-b925-325517351542)

你好，我也遇到了同样的问题，输出的全是黑白图，请问解决了吗

**Comment by qianxingyucode** [comment]
@Mr-Ma-yikun  Hi, I have solved the problem. I have tried to contact you, but failed. You can email me to get more information.

**Comment by snehalstomar** [comment]
@Treeboy2762  @qianxingyucode, could you please share the steps you took to solve the problem?
